### DISCUSSION TITLE
Custom categories

### DISCUSSION THREAD

**Discussion by Erdo987** [apohl79/audiogridder#329] [Ideas]
Hallo Andreas,
Ich hätte gern einen Wunsch.
Ich hätte gern in der plugin Liste selber Ordner erstellen,damit ich die Instrumente und Effekte selber kategorisieren kann,und meine Favoriten erstellen.
Ist es machbar? Oder gibt es schon so eine Möglichkeit?

Danke.

Mit freundlichen Grüßen

**Comment by apohl79** [comment]
(keeping this in english)

Custom categories are not supported right now, as AG uses the categories of the underlying plugin formats. It is doable though. So I might be able to look at this at some point in the future. I created a feature request issue for this.
